[2024-06-28T22:05:19.067+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-28T22:05:19.089+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: NYC_Taxi_pipeline.data_delivery manual__2024-06-28T22:05:15.621346+00:00 [queued]>
[2024-06-28T22:05:19.098+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: NYC_Taxi_pipeline.data_delivery manual__2024-06-28T22:05:15.621346+00:00 [queued]>
[2024-06-28T22:05:19.098+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-28T22:05:19.110+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): data_delivery> on 2024-06-28 22:05:15.621346+00:00
[2024-06-28T22:05:19.118+0000] {standard_task_runner.py:63} INFO - Started process 403 to run task
[2024-06-28T22:05:19.121+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'NYC_Taxi_pipeline', 'data_delivery', 'manual__2024-06-28T22:05:15.621346+00:00', '--job-id', '143', '--raw', '--subdir', 'DAGS_FOLDER/NYC_Taxi_pipeline.py', '--cfg-path', '/tmp/tmpwzk9bb2m']
[2024-06-28T22:05:19.123+0000] {standard_task_runner.py:91} INFO - Job 143: Subtask data_delivery
[2024-06-28T22:05:19.167+0000] {task_command.py:426} INFO - Running <TaskInstance: NYC_Taxi_pipeline.data_delivery manual__2024-06-28T22:05:15.621346+00:00 [running]> on host 79a59ce16f65
[2024-06-28T22:05:19.468+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Mahmoud' AIRFLOW_CTX_DAG_ID='NYC_Taxi_pipeline' AIRFLOW_CTX_TASK_ID='data_delivery' AIRFLOW_CTX_EXECUTION_DATE='2024-06-28T22:05:15.621346+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-28T22:05:15.621346+00:00'
[2024-06-28T22:05:19.469+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-28T22:05:19.480+0000] {NYC_Taxi_pipeline.py:54} INFO - data_delivery started
[2024-06-28T22:05:19.481+0000] {Load_Data.py:20} INFO - Loading location Dimension ...
[2024-06-28T22:05:19.488+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-28T22:05:19.493+0000] {Load_Data.py:36} INFO - Existing records columns: ['locationid', 'borough', 'zone', 'service_zone', 'start_date', 'end_date', 'active_flag', 'version']
[2024-06-28T22:05:19.632+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-28T22:05:19.634+0000] {Table_creation.py:51} INFO - Loading data into table DimLocation...
[2024-06-28T22:05:19.637+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-28T22:05:19.639+0000] {Table_creation.py:64} INFO - insert new row to dimlocation
[2024-06-28T22:05:19.640+0000] {Table_creation.py:64} INFO - insert new row to dimlocation
[2024-06-28T22:05:19.641+0000] {Table_creation.py:64} INFO - insert new row to dimlocation
[2024-06-28T22:05:19.642+0000] {Table_creation.py:66} INFO - Data loaded successfully into table DimLocation With 3 Row
[2024-06-28T22:05:19.642+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-28T22:05:19.642+0000] {Load_Data.py:90} INFO - DimLocation dimension loaded successfully
[2024-06-28T22:05:19.643+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-28T22:05:19.643+0000] {Load_Data.py:98} INFO - loading DimRate Dimension ...
[2024-06-28T22:05:19.645+0000] {Table_creation.py:30} INFO - Loading data into table DimRate...
[2024-06-28T22:05:19.648+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-28T22:05:19.655+0000] {Table_creation.py:45} INFO - Data loaded successfully into table DimRate With 6 Row
[2024-06-28T22:05:19.656+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-28T22:05:19.656+0000] {Load_Data.py:101} INFO - DimRate_df dimension loaded successfully
[2024-06-28T22:05:19.656+0000] {Load_Data.py:116} INFO - loading payment dimension....
[2024-06-28T22:05:19.659+0000] {Table_creation.py:30} INFO - Loading data into table DimPayment...
[2024-06-28T22:05:19.662+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-28T22:05:19.667+0000] {Table_creation.py:45} INFO - Data loaded successfully into table DimPayment With 6 Row
[2024-06-28T22:05:19.668+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-28T22:05:19.668+0000] {Load_Data.py:119} INFO - payment_df dimension loaded successfully
[2024-06-28T22:05:19.668+0000] {Load_Data.py:11} INFO - loading Dimvendor Dimension ...
[2024-06-28T22:05:19.671+0000] {Table_creation.py:30} INFO - Loading data into table DimVendor...
[2024-06-28T22:05:19.674+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-28T22:05:19.679+0000] {Table_creation.py:45} INFO - Data loaded successfully into table DimVendor With 2 Row
[2024-06-28T22:05:19.680+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-28T22:05:19.680+0000] {Load_Data.py:14} INFO - Dimvedor_df dimension loaded successfully
[2024-06-28T22:05:19.680+0000] {Load_Data.py:107} INFO - loading trip_type dimension....
[2024-06-28T22:05:19.682+0000] {Table_creation.py:30} INFO - Loading data into table DimTrip_type...
[2024-06-28T22:05:19.685+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-28T22:05:19.691+0000] {Table_creation.py:45} INFO - Data loaded successfully into table DimTrip_type With 2 Row
[2024-06-28T22:05:19.691+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-28T22:05:19.692+0000] {Load_Data.py:110} INFO - trip_df dimension loaded successfully
[2024-06-28T22:05:19.692+0000] {Load_Data.py:125} INFO - loading fact table ....
[2024-06-28T22:05:23.269+0000] {Table_creation.py:30} INFO - Loading data into table Fact_Trip...
[2024-06-28T22:05:23.272+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-28T22:08:38.259+0000] {Table_creation.py:45} INFO - Data loaded successfully into table Fact_Trip With 618792 Row
[2024-06-28T22:08:38.260+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-28T22:08:43.500+0000] {Load_Data.py:147} INFO - Fact_df  loaded successfully
[2024-06-28T22:08:43.530+0000] {NYC_Taxi_pipeline.py:61} INFO - data_delivery done
[2024-06-28T22:08:43.530+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-28T22:08:43.531+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-28T22:08:43.540+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=NYC_Taxi_pipeline, task_id=data_delivery, run_id=manual__2024-06-28T22:05:15.621346+00:00, execution_date=20240628T220515, start_date=20240628T220519, end_date=20240628T220843
[2024-06-28T22:08:43.586+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-28T22:08:43.601+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-28T22:08:43.603+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
